pulsar-common/src/main/java/org/apache/pulsar/common/naming/TopicName.java

```diff
@@ -170,8 +170,7 @@ private TopicName(String completeTopicName) {
                 throw new IllegalArgumentException("Invalid topic name: " + completeTopicName);
             }
 
-
-            if (localName == null || localName.isEmpty()) {
+            if (StringUtils.isBlank(localName)) {
                 throw new IllegalArgumentException("Invalid topic name: " + completeTopicName);
```

Comment:
It could be useful to mention that that the name was null or blank.

Reply:
I think that `localName` can never be null here.

What do you think of this?
```java
if (StringUtils.isBlank(localName)) {
    throw new IllegalArgumentException(String.format("Invalid topic name: %s. Topic local name must not"
                    + " be blank.", completeTopicName));
}
```

Reply:
LGTM

Reply:
Updated it.